wpeverest/everest-forms
# Fix - Email subject while using select field smart tag

### All Submissions:






### Changes proposed in this Pull Request:



This PR solves the bug with Email Subject while using the smart tag of the select field.

Closes: https://wordpress.org/support/topic/dropdown-content-not-displaying/

### How to test the changes in this Pull Request:

1. Create or Edit a form.
2. Go to the form settings and then Email then on Subject please select smart tag of a Select Field.
3. Then submit the form and then see the Email Subject before this PR the email subject was showing Array.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Email subject while using select field smart tag
